### DISCUSSION TITLE
Test  Installer for Windows

### DISCUSSION THREAD

**Discussion by LinKayser** [leap71/PicoGK#9] [General]
I packaged a first installer for Windows today. More work in the next couple of days, but if anyone wants to test it, here it is. It installs the PicoGK runtime files, so you can use them from C#. 

I will do some more polishing and also include a C# demo project, but let me know if this works for you.

Best,
Lin

[PicoGKInstaller_1.0.0_2023-11-15.zip](https://github.com/leap71/PicoGK/files/13365565/PicoGKInstaller_1.0.0_2023-11-15.zip)

**Comment by CorrieVS** [comment]
Hi Lin,
I ran this successfully on Windows 11, VS 2022, .NET 8.0.  Quick and easy 👍 
~Corrie

**Comment by vinit-AM** [comment]
> I packaged a first installer for Windows today. More work in the next couple of days, but if anyone wants to test it, here it is. It installs the PicoGK runtime files, so you can use them from C#.
> 
> I will do some more polishing and also include a C# demo project, but let me know if this works for you.
> 
> Best, Lin
> 
> [PicoGKInstaller_1.0.0_2023-11-15.zip](https://github.com/leap71/PicoGK/files/13365565/PicoGKInstaller_1.0.0_2023-11-15.zip)

not super familiar with code? would you mind little more support to set-up
